map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -343,18 +344,18 @@ class MapFragment : BaseViewModelFragment<MapViewModel>() {
                         map.animateCamera(
                                 CameraUpdateFactory.newLatLngZoom(
                                         centerOfMuseumOnMap,
-                                        ZOOM_MIN
+                                        ZOOM_INITIAL
                                 )
                         )
                     } else {
                         // Make sure we can see all of the specified positions
-                        map.moveCamera(CameraUpdateFactory
-                                .newLatLngBounds(
-                                        LatLngBounds.builder()
-                                                .includeAll(bounds)
-                                                .build(),
-                                        0
-                                ))
+                        val lanLngBounds = LatLngBounds.builder().includeAll(bounds).build()
```

Comment:
uh. lanLngBounds? What do you think of making this into a `val` for the center? Or rename to `areaOfInterestBounds`?

Reply:
That works!